wagtail/images/templates/wagtailimages/images/url_generator.html

```diff
@@ -35,8 +35,25 @@
         {% endpanel %}
 
         {% trans "URL" as heading %}
-        {% panel id="url" icon="link" heading=heading %}
-            <textarea class="w-image-url-generator__url" id="result-url" aria-labelledby="url-heading" rows="1" data-controller="w-action" data-action="focus->w-action#select"></textarea>
+        {% panel id="url" heading=heading %}
+            <div data-controller="w-clipboard w-clone" role="group" aria-labelledby="clipboardGroupLabel" data-w-clone-clear-delay-value="3500" data-action="w-clipboard:start->w-clone#clear w-clipboard:copied->w-clone#add w-clipboard:error->w-clone#add">
```

Comment:
```suggestion
            <div data-controller="w-clipboard w-clone" data-w-clone-clear-delay-value="3_500" data-action="w-clipboard:start->w-clone#clear w-clipboard:copied->w-clone#add w-clipboard:error->w-clone#add">
```

1. No need to use aria-labelledby here, we already have this within a panel that has a heading, additionally `div` elements will not normally even use this.
2. No need for `role=group` we are just using this div to manage attributes and maybe styles, so it's fine to be an element with no semantic meaning. It's just structural.
3. Stimulus lets us use underscore numbers, so let's use `3_500` for more readability. Although, are you sure you want to do this? My understanding is that the value will be delayed in clearing.
4. As noted in other feedback, I think `w-clipboard:copy` is probably a better name than `start`, I know I suggested it originally.


----

Note: another way to manage these attributes would be to pass them in as a Dict to `attrs` in the `panel`. However, this may get a bit verbose as we would have to build the attributes outside (in the Python/Django view).